app/services/generate_bill_run_summary.service.js

```diff
@@ -0,0 +1,67 @@
+'use strict'
+
+/**
+ * @module GenerateBillRunSummaryService
+ */
+
+const Boom = require('@hapi/boom')
+
+const { BillRunModel } = require('../models')
+
+class GenerateBillRunSummaryService {
```

Comment:
Could we rename this to

```suggestion
class GenerateBillRunService {
```

This will require a few changes elsewhere but it will mean it's consistent with our dropping of the term 'summary' in other places.